Adds Multistep Forms, Sidebar, and Basic Button

jvsluis opened the pull request:
Closes #11 
Closes #16 
And is linked to #22 however, there are no animations so I'm not linking the issue for closure until thats discussed

Changes:
Adds the multistep form and the multistep pages
Adds the skeleton of the form to both the browse and create page Added some more colour and styling to the application Everything is still rather bare bones and will need to be styled according to the spec Button is currently just an example of a simple component, however another pull should be done to give it proper styling

jvsluis committed:
add: multistep forms, sidebar, and basic button

Adds the multistep form and the multistep pages
Adds the skeleton of the form to both the browse and create page
Added some more colour and styling to the application
Everything is still rather bare bones and will need to be styled according to the spec
Button is currently just an example of a simple component, however another pull should be done to give it proper styling

Jasper-Nelligan commented on app/src/components/button.js:
```diff
@@ -0,0 +1,5 @@
+export default function Button({text}) {
+  return (
+    <button className="">{text}</button>
```
Is it intentional to have className be an empty string?

Jasper-Nelligan commented on app/src/components/multistepform.js:
```diff
@@ -0,0 +1,63 @@
+  }
+
+  return (
+    <div className="flex h-screen rounded-lg border border-gray-300 h-fit min-h-96 relative p-4">
```
There's a lot of long classNames in this file. Is it possible to extract these to a css file or something? I have no idea how tailwind works so I'm just wondering

jvsluis replied:
Yeah this is a tailwind thing. You're supposed to use their classes instead of direct css. The tailwind classes are designed to handle responsive behavior by default. Whereas with straight css you'd need to create manual media queries.

Jasper-Nelligan replied:
Hmm okay, that's something I'll have to learn to deal with then. Thanks for the clarification

Jasper-Nelligan commented on app/src/components/multistepformpage.js:
```diff
@@ -0,0 +1,7 @@
+export default function MultiStepFormPage({children, title}) {
+  return (
+    <div>
+      {children}
+    </div>
```
Is the div necessary here?

jvsluis replied:
React components need a parent. However it can be replaced with <> </> (a null parent). I can change that.

jvsluis committed:
fix: changed div to empty, added default btn class

Changed div to empty tag in multistepformpage
Added default button styles for Button component. This should be updated to match exact spec.
